neptune/management/exceptions.py

```diff
@@ -85,14 +85,15 @@ class UserNotExistsOrWithoutAccess(ManagementOperationFailure):
     code = 9
     description = (
         'User "{user}" does not exist or has no access to project "{project}". '
-        "If the project visibility is set to workspace, a user cannot be added or removed."
+        "If the project visibility is set to 'workspace', a user cannot be added or removed."
```

Comment:
Is the wording "added or removed" correct? As far as I can tell, this exception is raised only when you try to remove a user from a project. Could you confirm whether the removal will fail in the following scenarios:

1. The user was not found among the members of a private project
2. The project's visibility is "workspace" or "public"